content/docs/introducing-jsx.md

```diff
@@ -1,36 +1,36 @@
 ---
 id: introducing-jsx
-title: Introducing JSX
+title: Giới thiệu JSX
 permalink: docs/introducing-jsx.html
 prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+Xem xét khai báo biến dưới đây:
 
 ```js
 const element = <h1>Hello, world!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Cú pháp thẻ kì cục trên không phải là chuỗi cũng như HTML.
```

Comment:
Từ “kì cục” không nên dùng trong văn viết cũng như chưa sát với nghĩa gốc cho lắm. Mình gợi ý nên dùng từ: “thú vị” hoặc “kì lạ”.

Đoạn “không phải là chuỗi cũng như HTML” tuy dễ hiểu cho dev chuyên nghiệp, nhưng với người mới bắt đầu tìm hiểu React thì mình có thể nên thêm một số từ để bổ nghĩa, giúp cho bản văn dễ hiểu cho mọi đối tượng:
Gợi ý của mình là: “không phải là một chuỗi kí tự hoặc một thẻ HTML”

Reply:
> This funny tag syntax is neither a string nor HTML.
> Cú pháp thẻ kì lạ trên không phải là một chuỗi kí tự hoặc một thẻ HTML.
Kết hợp từ ý của em @nampdn, chị thấy hợp lí.

Reply:
It should be "Cú pháp thẻ này không phải là một chuỗi kí tự cũng không phải là một thẻ HTML".